docker/php-5.6/xdebug.ini

```diff
@@ -0,0 +1,5 @@
+;zend_extension=xdebug.so
+xdebug.remote_autostart=1
+xdebug.remote_enable=1
+xdebug.remote_host=host.docker.internal
+xdebug.remote_port=9003
```

Comment:
Images that use Xdebug 2 (PHP < 7.3) use the port 9003 instead of the default 9000 because on Xdebug 3 the default port has been changed to 9003. This removes the need to change the port to listen in e.g. PhpStorm everytime you switch between versions of PHP.

Reply:
maybe we don't need xdebug for 5.6, which we gonna drop soon ?

Reply:
It's exactly the same config than for PHP 7.0, 7.1 and 7.2, so PHP 5.6 does not specifically make the whole setup more complicated.